slackdump/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .command import main
```

Comment:
@UmangThapliyal @ManishaBayya  Here `.` is not required. We can directly access `from command import main`. Follow the same thing with below lines also.

Reply:
In Python3 to import modules we should include . else it will show error

Reply:
@ManishaBayya  @UmangThapliyal https://docs.python.org/3/tutorial/modules.html please check this. User-defined modules need not be called with, `.`, also if this is the case, then please let us know, why `.` is required in python3 and not in python2

Reply:
@shivam-deepcompute Can u please refer the below link.
https://stackoverflow.com/questions/12172791/changes-in-import-statement-python3